### DISCUSSION TITLE
Control WiFi accessory with IKEA switch?

### DISCUSSION THREAD

**Discussion by darkner258** [Koenkk/zigbee2mqtt#14106] [Devices]
Hi everyone, how do I do that?

I have an IKEA switch which I would love to use to control my Vocolinc Aroma Fan (simple on/off) would do. Any suggestions on this, please?

**Comment by DubhAd** [comment]
You would need some other software to handle that, something like Home Assistant, OpenHAB, etc

**Reply by darkner258** [reply]
Or possibly Node Red? I have that up and running, just dont know how to set it up

**Reply by DubhAd** [reply]
Node Red can certainly do it. I don't use it myself, but I know that others do for exactly this kind of thing.
